app/src/main/java/org/covidwatch/android/ble/BluetoothManager.kt

```diff
@@ -83,7 +83,7 @@ class BluetoothManagerImpl(
     private fun foregroundNotification(): Notification {
         createNotificationChannelIfNeeded()
 
-        val notificationIntent = Intent(app, MainActivity::class.java)
+        val notificationIntent = Intent(app, TempActivity::class.java)
```

Comment:
`TempActivity` was a temporary measure while we had `MainActivity`. It's better if you just rename `TempActivity` to `MainActivity` and the code that depended on the old `MainActivity` won't simply change. 

ie we wouldn't need to rename all references to `MainActivity` to `TempActivity`

Reply:
Clean all code related to this, including `AndroidManifest.xml`

Reply:
Done.